job_runner/reporting/event.py

```diff
@@ -68,6 +74,7 @@ def report(self, msg):
         if self._state not in self._transitions or not isinstance(
             msg, self._transitions[self._state]
         ):
+            logger.debug("Illegal state transition")
```

Comment:
Why isn't there information about the exception here? It contains useful information.

Reply:
It isn't caught, but it is logged (to disk).